modules/dropdown/SimpleMultipleDropdown.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import { WithStylesProps } from "@diana-ui/types";
+import { withStyles } from "@diana-ui/base";
+import BaseMultipleDropdown, {
+  IMultipleProps,
+  multipleStylesheet
+} from "./BaseMultipleDropdown";
+import { IItem } from "./Dropdown";
+
+export interface ISimpleMultipleDropDownProps<T extends IItem>
+  extends IMultipleProps<T> {}
+
+const StylesBaseMultipleDropdown = BaseMultipleDropdown.extendStyles(
+  multipleStylesheet
+);
+
+const SimpleMultipleDropdown: React.FC<
+  ISimpleMultipleDropDownProps<IItem> & WithStylesProps
+> = ({ onItemsSelected, items, selectedItems, ...props }) => {
+  const [draftItems, setDraftItems] = useState([...selectedItems]);
+  const isAllSelected = items.length === draftItems.length;
+
+  const onItemClicked = (item: IItem) => {
+    let newItems = [...draftItems];
+    const selected = newItems.find(i => i.id === item.id) !== undefined;
+    if (selected) {
+      newItems = newItems.filter(i => i.id !== item.id);
+    } else {
+      newItems.push(item);
+    }
+    setDraftItems(newItems);
+  };
+  const onAllButtonClicked = () => {
+    if (isAllSelected) {
+      setDraftItems([]);
+    } else {
+      setDraftItems([...items]);
+    }
+  };
+
+  const onClose = () => {
+    onItemsSelected([...draftItems]);
```

Comment:
A comment here explaining why we only want to update the selected items `onClose` because of the ordering would be helpful! 🙏